client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/FormDeleteButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react';
+
+export const FormDeleteButton = (props: {
```

Comment:
I'm not too sure why it's happening but it seems like deleting a showing doesn't work. The modal pops up saying its success but the showing itself just stays there. Is this intended?

Reply:
It seems to work for me?

Reply:
I haven't been getting this? Could you try rebuilding and seeing if this still happens?